docs/BUILD

```diff
@@ -36,6 +36,8 @@ PAGES = {
         "rust_binary",
         "rust_benchmark",
         "rust_test",
+        "error_format",
```

Comment:
Sure I can remove it from this PR.

Reply:
> @UebelAndre is there a way to not have to check in generated docs, but build them on the CI? Have you chatted with Bazel folks about this? Are all the Bazel rules checking in generated docs?

There are a few stragglers which are not being generated if I recall but for the most part, yeah, all major rules are hooked up to generate docs. If CI can run a script and commit something back to the repo then we can have CI update docs no problem. Simply run `./docs/update_docs.sh`. I think this'd be a fantastic improvement. In general, if there's a way to run arbitrary scripts, we can test to prevent generated docs from being modified and forcing users to update the source of truth (which has happened from time to time and documentation was stomped).

Reply:
Split to https://github.com/bazelbuild/rules_rust/pull/924